src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -3661,8 +3731,8 @@ int MacroAssembler::max_trampoline_stub_size() {
 }
 
 int MacroAssembler::static_call_stub_size() {
-  // (lui, addi, slli, addi, slli, addi) + (lui, addi, slli, addi, slli) + jalr
-  return 12 * NativeInstruction::instruction_size;
+  // (lui, addi, slli, addi, slli, addi) + (lui + lui + ssli + add) + jalr
```

Comment:
Instead of `ssli`, shouldn't it be `slli`?

Reply:
Yes, for some reason I keep writing ssli instead slli :)